__tests__/api.test.js

```diff
@@ -55,3 +55,38 @@ describe("GET /api", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id", () => {
+  it("status: 200, responds with an article object", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body: { article } }) => {
+        expect(article.article_id).toBe(1);
+        expect(article.title).toBe("Living in the shadow of a great man");
+        expect(article.topic).toBe("mitch");
+        expect(article.body).toBe("I find this existence challenging");
+        expect(article.votes).toBe(100);
+        expect(article.created_at).toBe("2020-07-09T20:11:00.000Z");
+        expect(article.article_img_url).toBe(
+          "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"
+        );
+      });
```

Comment:
Using toHaveProperty isn't bad, but there is better Jest Matchers we can use for more robust and extendable testing, take a look into toMatchObjects and expect.objectContaining.